modules/webm/video_stream_webm.cpp

```diff
@@ -205,7 +205,10 @@ float VideoStreamPlaybackWebm::get_playback_position() const {
 	return video_pos;
 }
 void VideoStreamPlaybackWebm::seek(float p_time) {
-	WARN_PRINT_ONCE("Seeking in Theora and WebM videos is not implemented yet (it's only supported for GDNative-provided video streams).");
+	//WARN_PRINT_ONCE("Seeking in Theora and WebM videos is not implemented yet (it's only supported for GDNative-provided video streams).");
+
```

Comment:
We prefer removing code entirely instead of commenting it out. You don't need to change it in this PR, but remember to do this on the `3.x` PR when opening it :slightly_smiling_face:

Reply:
Got it, Thanks! 😃